packages/adapter-sveltekit/test/plugin-project-init.test.ts

```diff
@@ -1038,10 +1038,23 @@ describe("root layout file", () => {
 
 		expect(contents).toMatchInlineSnapshot(`
 			"<script>
+			  import { isFilled, asImageSrc } from \\"@prismicio/client\\";
 			  import { PrismicPreview } from \\"@prismicio/svelte/kit\\";
+			  import { page } from \\"$app/stores\\";
 			  import { repositoryName } from \\"$lib/prismicio\\";
 			</script>
 
+			<svelte:head>
+			  <title>{$page?.data.title}</title>
```

Comment:
❓ No check that title is defined both `title` and `meta`?

Reply:
I think it's just `page` that might be undefined and `title` tag is mandatory-ish. Not sure if we want to do that check for the `og:title` meta 🤔